core/tests/test_util_jobs.py

```diff
@@ -66,21 +74,44 @@ class ExpCopiesMRJobManager(
 
     @classmethod
     def _get_continuous_computation_class(cls):
+        """Returns the ExpCopiesAggregator class associated with this MapReduce
+        job.
+        """
         return ExpCopiesAggregator
 
     @classmethod
     def entity_classes_to_map_over(cls):
+        """Returns the ExplorationModel domain object."""
         return [exp_models.ExplorationModel]
 
     @staticmethod
     def map(item):
+        """Yields the string containing exploration id with the copy number of
```

Comment:
What's a "copy number"? Might need a better explanation.

Reply:
Copy number here means the count indicating  the current copy of the exploration with given id. I've updated the docstrings as well.